etcd/etcd-env.sh

```diff
@@ -6,11 +6,11 @@ if test x$NAME == x; then
 fi
 export ETCD_NAME=${ETCD_NAME:-$HOSTNAME}
 export ETCD_DATA_DIR=/var/lib/etcd/${NAME}.etcd
```

Comment:
You could create a variable in ``manifest.json`` called ``DATA_DIR`` or something similar. ``DATA_DIR`` would then be used in the templates and shell script. The default could be ``/var/lib/$NAME``. If ``DATA_DIR`` is overridden then it would point to the provided location instead.